recipe/meta.yaml

```diff
@@ -31,7 +31,7 @@ requirements:
     - python
     - py6s
     - gdal 2.2.*
```

Comment:
I think you need to pin to `gdal 2.1.*` to match `rsgislib` etc. I think this is why the builds are failing....